src/azure-cli/HISTORY.rst

```diff
@@ -55,6 +55,10 @@ Release History
 
 * az deployment create: Add --handle-extended-json-format parameter to support multiline and comments in json template.
 
+**AppConfig**
+
+* Added Feature Management CLI Support
```

Comment:
Can you change it to "Add appconfig feature command group to manage feature flags stored in an App Configuration." or something like this to have easy understanding?